### PR TITLE
feat(deps): Add Nextcloud 30 support

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@ For this app to be effective, the ClamAV virus definitions should be kept up to
 	<screenshot>https://raw.githubusercontent.com/nextcloud/files_antivirus/master/screenshots/1.png</screenshot>
 
 	<dependencies>
-		<nextcloud min-version="25" max-version="28" />
+		<nextcloud min-version="25" max-version="29" />
```
Comment (melroy89) [inline appinfo/info.xml:47]: PR title says Nextcloud 30. Here is max version 29.. Is that on purpose?

**Review by melroy89 — APPROVED** [review]
Nice. It also have nextcloud 30 support now.

**Comment by MatthiasKuehneEllerhold** [comment]
Whats the status of this PR?

**Comment by MatthiasKuehneEllerhold** [comment]
Let me rephrase the question: now that NC 30 is released, when are you planning to merge this PR so we can have AV functionality back in NC?

**Comment by melroy89** [comment]
> Let me rephrase the question: now that NC 30 is released, when are you planning to merge this PR so we can have AV functionality back in NC?

I can still enable the plugin in NC30. And it does still seem to work.

**Comment by st3iny** [comment]
Fixed in d713fd051fd2da71dc3711b9de924e666ada40d
